types/transaction.go

```diff
@@ -14,9 +14,9 @@ type Transaction struct {
 	To       *Address
 	Value    *big.Int
 	Input    []byte
-	V        []byte
-	R        []byte
-	S        []byte
+	V        big.Int
```

Comment:
Shouldn't this be a `*big.Int`?

Reply:
It makes more sense to be *big.Int. But I was unsure how to refactor the sender logic then.
The Previous type for these attributes was []byte and if we create Transaction object without them they would still get assigned as null values. These attributes (V, R, S) we use to encode signature (even if they are null values). If we use pointers we would not be able to encode signature because the default value is nill. That's why I used reference because null values are the same. So I do not change sender/signature logic. 

The solution that was implemented before switching to reference was putting guard before encodeSignature to define attributes if they do not exist. 
```
	if tx.S == nil {
		tx.S = big.NewInt(0)
	}
```
But I find this solution messy.

The second thing I wanted to try was changing overall logic but was not comfortable doing it, not knowing the code base well enough.

Reply:
I've changed these attributes to pointers.